packages/ckeditor5-core/src/multicommand.js

```diff
@@ -5,6 +5,8 @@
 
 import Command from './command';
 
+import insertByPriority from '@ckeditor/ckeditor5-utils/src/insertbypriority';
```

Comment:
This name does not explain the purpose of this util. Is it some sort of priority queue?